documentation-src/metalsmith/content/concepts.md

```diff
@@ -217,6 +217,27 @@ parameters object and its api to create new states that we can serialize.
 
 ![Search states](images/concepts/State is immutable.svg)
 
+### Derivations of the helper: multi-queries
```

Comment:
Is multi index search one of the main usecases of this feature? Maybe we could name drop multi index somewhere then?